docs: DOC-1929: GPU > Supported Machine Types GCP k8s Fix (#7234)
spectrocloud/librarium: docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md

> - Autoscaling is not supported for GKE clusters.

Reviewer: ```suggestion
- GKE clusters do not support autoscaling.
```

Reviewer: ```suggestion
- The system does not support autoscaling on GKE clusters.
```

Reviewer: ```suggestion
- Autoscaling functionality is unavailable for GKE clusters.
```
So I provided several options to avoid passive voice for the next limitation, and then I noticed this one :)

Author: Sveta offered great suggestions, but after a discussion, we've decided to keep it in passive voice. Style guide has been updated to reflect that passive voice is preferable for limitations.

> - GPU machine types cannot be used to configure node pools.

Reviewer: ```suggestion
- You cannot use GPU machine types to configure node pools.
```

Reviewer: ```suggestion
- Using GPU machine types to configure node pools is not possible.
```

Reviewer: ```suggestion
- The system does not allow configuring node pools with GPU machine types.
```
I provided several options to avoid passive voice
